### DISCUSSION TITLE
Currency of some securities changed from GBp to GBP (yahoo)

### DISCUSSION THREAD

**Discussion by JPFrancoia** [ghostfolio/ghostfolio#2976] [General]
**Bug Description**

I don't think it's a bug with Ghostfolio, but the price of some securities seems to be given in GBP now, instead of GBp (so it's 100x smaller) on Yahoo. For example: https://finance.yahoo.com/quote/0P00018XAR.L/history?p=0P00018XAR.L

![screen-2024-02-08-17-41-20](https://github.com/ghostfolio/ghostfolio/assets/2399090/9f8a7dcd-f446-4207-96e2-d5c8c1acd9ba)

This means that the value of my portfolio seems to have dropped big time. This is an example for one of my investments:

![screen-2024-02-08-17-42-03](https://github.com/ghostfolio/ghostfolio/assets/2399090/a8297e6c-adcd-4fef-b094-2ce2bfdad2ed)

It looks like the values are in GBP, but the data seems to be tagged as GBp.
Is there any way to prevent that? At the moment the numbers I'm getting are not usable. Is it possible to change data source?

**Comment by AlexCPU** [comment]
Seems like Yahoo finance have fixed their data.  I had to delete from the 1st Feb in the database and rescrape data to fix it.

**Comment by JPFrancoia** [comment]
Yep, it's all good now, thanks. They fixed it just a few hours I posted here
